apluslms_roman/backends/docker.py

```diff
@@ -75,6 +77,8 @@ def build(self, task, observer):
             observer.start_step(step)
             opts = self._run_opts(task, step)
             observer.manager_msg(step, "Running container {}:".format(opts['image']))
+            if isinstance(opts["command"], ChangesList):
```

Comment:
This is not a good place to handle this. I think the best place is in `BuildStep.__init__`: https://github.com/apluslms/roman/blob/f2ec26f6e41091d51d7304ded3706adfca231c28/apluslms_roman/backends/__init__.py#L51

Both env and cmd should be handled there. For example:

```
self.img = clean_image_name(img)
self.cmd = cmd if cmd is None or isinstance(cmd, str) else list(cmd)
self.env = {} if env is None else dict(env)
```

and remove all those actions from `from_config`.

Reply:
I'm assuming this also applies to everything in the milestone0.3_global_vars branch?

Reply:
I presume so.